components/tabs/use-imperative-tabs.ts

```diff
@@ -0,0 +1,15 @@
+import { useRef, RefObject } from 'react'
```

Comment:
the file name is better to be use-tabs-handle

Reply:
done

Reply:
done